### DISCUSSION TITLE
May I be invited to slack community?

### DISCUSSION THREAD

**Discussion by Sijie-Yang** [zepp-health/zeppos-forum#316] [Q&A] [answered]
Hi there my email address is sijie.1.yang@outlook.com. I find it difficult to join the slack without invitation through the link you share on developer official website.

**Comment by silver-zepp** [comment, marked as answer]
Hey @Sijie-Yang, please try using this link: https://bit.ly/zepp-invite-feb

**Reply by Sijie-Yang** [reply]
Thank you Silver!

**Comment by gingerbeardman** [comment]
The community is now on Discord: https://zeppos.dev/amazfit-discord
